playwright-home-task-1/test-2-add-product-to-the-cart.spec.js

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+
+test('Verify Shopping Cart icon contains the number of products added', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+  await page.locator('[data-test="username"]').fill('standard_user');
```

Comment:
You can use `page.getByTestId` if you do this step https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute